apps/docs/package.json

```diff
@@ -27,12 +27,12 @@
   },
   "devDependencies": {
     "@hitachivantara/uikit-uno-preset": "*",
-    "@types/node": "^20.11.6",
-    "@types/react": "^18.2.43",
-    "@unocss/postcss": "^0.58.5",
+    "@types/node": "^22.9.0",
```

Comment:
Why?
We're on `node@20` on the root, which is already two majors ahead of the versions our CI run (and that we officially support)

Reply:
you're right. i just bulk bumped, my bad.